docs/source/guide/testplan.rst

```diff
@@ -282,19 +257,17 @@ Kiwi TCMS offers functionality to export test cases via `TCMS API <https://kiwit
 .. |The Search menu| image:: ../_static/Click_Search.png
 .. |The Search Plan screen| image:: ../_static/Planning_Home.png
 .. |Click column to sort by.| image:: ../_static/Sort_By_Column.png
-.. |The Clone plan button| image:: ../_static/Plan_Actions.png
+.. |The Clone plan button| image:: ../_static/Plan_Actions_Clone.png
 .. |The Clone Test Plan screen| image:: ../_static/Clone_Details.png
-.. |The Edit plan button| image:: ../_static/Plan_Actions.png
+.. |The Edit plan button| image:: ../_static/Plan_Actions_Edit.png
```

Comment:
These buttons do not get highlighted. I believe the image is incorrect. 

There is no need to specific images for every action button. 1 image (like it was before) without the highlight should be enough. Then you don't have to change the name of this file.

Reply:
One image, without highlighted button.